devicetypes/axis/axis-gear-st.src/axis-gear-st.groovy

```diff
@@ -199,27 +200,32 @@ def off() {
 //Command to set the blind position (%) and log the event
 def setLevel(value, rate=null) {
 	log.info "setLevel ($value)"
-	
+
```

Comment:
Should call `setShadeLevel` from here

Reply:
Haha Ooops. Pushed the commit a little quickly.